spec/controllers/market_lists_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe MarketListsController, type: :controller do
+  describe 'GET #index' do
+    it 'assigns @market_lists ordered by market_date' do
+      market_list1 = MarketList.create(name: 'Lista 1', market_date: Date.today)
```

Comment:
It's a good practice to use let instead of variables on test, like this:

`let(:market_list1) { MarketList.create(name: 'Lista 1', market_date: Date.today) }`